telemetry/specs/INT.mdk

```diff
@@ -833,7 +905,7 @@ INT Shim Header for VXLAN-GPE, destination INT type:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-INT Metadata Header and Metadata Stack, 
+INT Metadata Header and Metadata Stack,
 followed by encapsulated Ethernet payload:
 `
     0                   1                   2                   3
```

Comment:
Please revise the header examples with the newly added M bit.

Reply:
Done.